analyzers/src/SonarAnalyzer.CFG/Helpers/RoslynHelper.cs

```diff
@@ -29,6 +29,11 @@ namespace SonarAnalyzer.CFG.Helpers
 {
     internal static class RoslynHelper
     {
+        private const int MinimalSupportedMajorVersion = 3;
+
+        public static bool RoslynVersionIsSupported(int minimalVersion = MinimalSupportedMajorVersion) =>
```

Comment:
Let's add `Cfg` into the name, because everything is `Roslyn` around here.
```suggestion
        public static bool IsRoslynCfgSupported(int minimalVersion = MinimalSupportedMajorVersion) =>
```